Makefile

```diff
@@ -20,8 +20,9 @@ CXXFLAGS := -Wall \
 	-Wmissing-include-dirs \
 	-Wno-unused-parameter \
 	-Wuninitialized \
-	-Wno-reorder
+	-Wno-reorder \
```

Comment:
This change isn't related to the pull request. I understand that this is to prevent merge conflicts upon adding more warning flags, but I don't intend to change them often and personally dislike the trailing backslash on the last line. The next time I make changes to this repo I'll enable many more warnings.